projects/nae-example-app/src/app/doc/public-task-view/public-task-view.component.ts

```diff
@@ -112,8 +116,13 @@ export class PublicTaskViewComponent extends AbstractTaskView implements AfterVi
 
     @ViewChild('header') public taskHeaderComponent: HeaderComponent;
 
-    constructor(taskViewService: TaskViewService) {
+    constructor(taskViewService: TaskViewService, publicTaskLoadingService: PublicTaskLoadingService) {
         super(taskViewService);
+        this.loading$ = combineLatest(taskViewService.loading$, publicTaskLoadingService.loading$).pipe(
+            map(sources => {
+                return !(!sources[0] && !sources[1]);
```

Comment:
would be more readable as `sources[0] || sources[1]`, but that doesn't seem to be equivalent. Is the expression correct?

Reply:
Yes, expression is correct. I update it.